src/layout/Footer/Footer.svelte

```diff
@@ -15,10 +15,6 @@
 			name: "TheOnlyTails",
 			link: "https://www.theonlytails.com/"
 		},
-		{
-			name: "Yair",
-			link: "https://github.com/yaichenbaum/"
-		},
 		{
```

Comment:
Why not update the URL instead of removing this entry?

Reply:
Yair asked me specifically on discord to remove it.